designTokenKeySorter.js

```diff
@@ -0,0 +1,190 @@
+/* eslint-disable no-console */
+const fs = require('fs').promises;
```

Comment:
[inform]: If we are satisfied with this functionality, it will be moved to Paragon CLI